math/lcm_sum.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file
+ * @brief An algorithm to calculate the sum of LCM: \f$\mathrm{LCM}(1,n) + \mathrm{LCM}(2,n) + .. + \mathrm{LCM}(n,n)\f$
+ * @details An algorithm to calculate the sum of LCM: \f$\mathrm{LCM}(1,n) + \mathrm{LCM}(2,n) + .. + \mathrm{LCM}(n,n)\f$ where \f$\mathrm{LCM}(i,n)\f$
+ * denotes the Least Common Multiple of the integers i and n. For n greater than or equal to 1.
+ * The value of the sum is calculated by formula:
+ * \f[
+ * \sum\mathrm{LCM}(i, n) = \frac{1}{2} \left[\left(\sum (d * \mathrm{ETF}(d)) + 1\right) * n\right]
+ * \f]
+ *
+ * @author [Chesta Mittal](https://github.com/chestamittal)
+ */
+
+#include <iostream> /// for std::cin and std::cout
+#include <cassert>  /// for assert
+#include <vector>   /// for std::vector
+
+/**
+ * @namespace math
+ * @brief Mathematical algorithms
+ */
+namespace math {
+    /**
+     * Function to compute sum of euler totients in sumOfEulerTotient vector
+     * @param num input number
+     * @returns int Sum of LCMs, i.e. ∑LCM(i, num) from i = 1 to num
+     */
+    int lcmSum(int num) {
```

Comment:
the output will quickly start growing. `int` may not be enough. Please use `uint64_t` for safety. This would also require appropriate changes in the code.
The suggestion below also limits the range of input numbers to 16-bit unsigned integers. 
```suggestion
    uint64_t lcmSum(uint16_t num) { 
```

Reply:
Changed.